docs/sources/style-guide/write-for-developers/index.md

```diff
@@ -181,14 +181,14 @@ In Markdown, in front of a placeholder, use an asterisk followed by a backtick.
 
 ````
 ```
-The following text is a placeholder: *`A_PLACEHOLDER`*.
+The following text is a placeholder: *`<A-PLACEHOLDER>`*.
```

Comment:
You need to update the sentence before the examples too.  It currently reads, "In Markdown, in front of a placeholder, use an asterisk followed by a backtick. At the end of the placeholder, use a backtick followed by an asterisk. For example, prefer the following raw markdown:"

Reply:
Good catch, thanks.